src/coord/src/client.rs

```diff
@@ -378,7 +378,7 @@ impl SessionClient {
             }
         }
 
-        fn datum_to_json(datum: &Datum, idx: usize, col_types: &[ColumnType]) -> serde_json::Value {
```

Comment:
the `idx` and `col_types` args were just not used except in the recursive call? That's a little rough.

Reply:
They were needed with the old decimal impl; when that got ripped out understandably this didn’t get simplified.